rclcpp/include/rclcpp/service.hpp

```diff
@@ -172,6 +172,7 @@ class Service : public ServiceBase
         (std::cerr << ss.str()).flush();
         rcl_reset_error();
       }
+      delete service_handle_;
```

Comment:
do you actually need that? I'd assume that `rcl_service_fini` cleans up all heap allocations associated with that `service_handle_` and the destructor deletes this instances after going out of scope. But maybe I am missing something here.